articles/introduction-to-sequalize-orm-for-nodejs/index.md

```diff
@@ -0,0 +1,215 @@
+
+We first set up a new npm package as shown below:
+
+```bash
+npm init -y
+```
+
+We then install Sequelize as shown below:
+
+```bash
+npm install sequelize --save
+```
+
+It is available via `npm` and `yarn`. The `--save` makes the dependency available in our `package.json` file. You'll then have to install the driver of your preferred database, for example:
+
+```bash
+npm install pg pg-hstore --save
+npm install mysql2 --save
+npm install mariadb --save
+npm install sqlite3 --save
+npm install tedious --save
+```
+
+### SQL database Basics
+Let's brush our knowledge on SQL before we get to Sequelize. Structured Query Language is a language that allows us to interact with databases in performing database queries. Pronunciation differs from `SQL` to `sequel` your choice. SQL only allows execution on relational databases.
+
+SQL enables us to perform operations abbreviated as `CRUD`; Create Read Update and Delete. To illustrate this we have two tables in our database `Employees`:
+
+Table `lego_people`
+
+|name   |age    |
+|-----  |----   |
+|james  |34     |
+|craig  |15     |
+|paul   |67     |
+|cynthia|55     |
+
+
+Table `lego_height`
+
+|name   |cm     |
+|-----  |----   |
+|james  | 4     |
+|craig  | 5     |
+|paul   | 7     |
+|cynthia| 5     |
+
+So if we want to `create` data in the `lego_people` table we:
+
+```sql
+INSERT INTO lego_people(name, age)
+VALUES ('Joe' ,12);
+```
+The `INSERT INTO` statement is used to create data in a database table. We pass in `name` and `age` as arguments to specify the properties we want to add.
+To `read` value from the `lego_people` table:
+
+```sql
+SELECT name FROM lego_people;
+```
+The `name` represents the column name in the` lego_people` table. The `SELECT` statement is used to read data from the database.
+
+To `update` a record in the `lego_height` table by making Paul 8cm tall:
+
+```sql
+UPDATE lego_height
+SET name = 'paul', height = 8;
+```
+
+We can also perform a `delete` operation from the database. For example, let's delete `paul` altogether using the following command:
+
+```sql
+DELETE FROM lego_height
+WHERE name = 'paul';
+```
+
+The `WHERE` clause specifies the data to be deleted.
+
+If for instance, we wanted to have the records of both age and height in one table we would use a `JOIN` statement to combine both tables:
+
+```sql
+SELECT name, age FROM lego_people
+LEFT JOIN lego_height USING (name);
+```
+
+This would result to this:
+
+|name   |age   |cm    |
+|-----  |----- |----- | 
+|james  |34    | 4    |    
+|craig  |15    | 5    |
+|paul   |67    | 7    |    
+|cynthia|55    | 5    |
+
+### Object Relational Mappers(ORM) Node.js Approaches
+Before we get started with Sequelize, let's get to understand what an ORM is and some of the supported ORMs in Node.js. Object Relational Mapping is a simplified way of converting data between relational databases and objects (Primrose katena, 2020).
```

Comment:
Is this `(Primrose katena, 2020)` some reference? If so, could you also add a link here.